update create magic url session doc (#2847)
appwrite/appwrite: docs/references/account/create-magic-url-session.md

Sends the user an email with a secret key for creating a session. When the user clicks the link in the email, the user is redirected back to the URL you provided with the secret key and userId values attached to the URL query string. Use the query string parameters to submit a request to the [PUT /account/sessions/magic-url](/docs/client/account#accountUpdateMagicURLSession) endpoint to complete the login process. The link sent to the user's email address is valid for 1 hour.

> If you are on a mobile device, you must explicitly provide the URL parameter and handle the redirection of that URL to your app so that users clicking on the email will be redirected to your app with userId and secret. You can call `updateMagicURLSession` using the obtained information to create a valid session for the user in your application.

Reviewer: `If you are on a mobile device` might be a bit misleading. Since also WebApps can be on a mobile device.

Author: What do you suggest?

Reviewer 2: How about:

> If you are building a mobile app, you must explicitly...

or:

> If you are not building a web app, you must explicitly...

> If you not building a web application, you must explicitly provide the URL parameter and handle the redirection of that URL to your app so that users clicking on the email will be redirected to your app with userId and secret. You can call `updateMagicURLSession` using the obtained information to create a valid session for the user in your application.

Reviewer 3: ```suggestion
If you are not building a web application, you must explicitly provide the URL parameter and handle the URL redirect to your app so users will be redirected to your app with their userId and secret. You can call `updateMagicURLSession` using the obtained information to create a valid session for the user in your application.
```
